src/test/java/com/exasol/bucketfs/jsonrpc/CreateBucketCommandIT.java

```diff
@@ -4,70 +4,46 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
 import static org.junit.jupiter.api.Assertions.*;
-import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
-import java.time.Duration;
-import java.time.Instant;
 import java.util.concurrent.TimeoutException;
 
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import com.exasol.bucketfs.*;
-import com.exasol.bucketfs.jsonrpc.CreateBucketCommand.CreateBucketCommandBuilder;
-import com.exasol.bucketfs.monitor.TimestampRetriever;
-import com.exasol.containers.ExasolDockerImageReference;
+import com.exasol.bucketfs.testutil.BucketCreator;
 
 @Tag("slow")
 // [itest->dsn~creating-new-bucket~1]
 class CreateBucketCommandIT extends AbstractBucketIT {
-    private static final String READ_PASSWORD = "READ_PASSWORD";
-    private static final String WRITE_PASSWORD = "WRITE_PASSWORD";
 
     @Test
     void testCreatingBucketWithCheckingCertificateThrowsException() throws BucketAccessException, TimeoutException {
-        assumeJsonRpcAvailable();
-
-        final String randomBucketName = getUniqueBucketName();
-
-        final CreateBucketCommandBuilder command = createCommandFactory(true).makeCreateBucketCommand() //
-                .bucketFsName(DEFAULT_BUCKETFS) //
-                .bucketName(randomBucketName) //
-                .isPublic(true) //
-                .readPassword(READ_PASSWORD) //
-                .writePassword(WRITE_PASSWORD);
-
-        final JsonRpcException exception = assertThrows(JsonRpcException.class, () -> command.execute());
+        final BucketCreator bucketCreator = bucketCreator().assumeJsonRpcAvailable();
```

Comment:
This refactoring is a bit out of scope of the current ticket. Next time I suggest to split it off into its own PR. That would also make reviewing easier.